CODEOWNERS

```diff
@@ -302,7 +302,7 @@
 /drivers/pwm/pwm_shell.c                  @henrikbrixandersen
 /drivers/pwm/*gecko*                      @sun681
 /drivers/pwm/*it8xxx2*                    @RuibinChang
-/drivers/regulator/*i2c*		  @danieldegrasse
+/drivers/regulator/*pmic*		  @danieldegrasse
```

Comment:
This PR has a setup based on the name "i2c", but then changes it later to use "pmic", here and elsewhere in this commit.

Please clean up the commits so the lines get added the way you want them the first time, without this type of change in a single PR.

Reply:
Fixed, thank you